### ISSUE TITLE
Handle window resizing

### ISSUE THREAD

**Issue by carlosdr02** [ocornut/imgui#4944]
Hi, I was looking for a way to know when the user has finished resizing a window (i.e., the user has released the mouse from border resizing control).
I already know how to know if the window has been resized; comparing the window size of the current frame with the previous frame, but I need to have a callback to when the user has stop resizing.

**Comment by ocornut** [comment]
Why do you need this?

**Comment by carlosdr02** [comment]
Because I need to recreate a Vulkan image with the new size but if I recreate it every frame that the user is in the middle of a resize operation I would just mess up the maximum number of Vulkan allocations.

**Comment by carlosdr02** [comment]
Yes, in Vulkan there is, which is usually 4096 on most drivers, so I was thinking... While the resize is happening, the image could be upscaled/downscaled, so it doesn't just stop the rendering, but also doesn't allocate device memory every frame, and once the resize has finished, the images would be recreated.

**Comment by ocornut** [comment]
You shouldn't need more than 2-3 in-flight framebuffer maximum... once finished rendering an old one should be freed anyway.

**Comment by carlosdr02** [comment]
Yeah, but the memory needed for the new one might be bigger/smaller depending on the new size of the viewport, hence a new size for an image.

**Comment by ocornut** [comment]
Yes but you shouldn't need more than 2-3.

Either way my best suggestion is you use `IsAnyItemActive()`.
You can also technically use `imgui_internal.h`, query `ImGui::GetActiveID()` and compare that value to all 8 values of `GetWindowResizeCornerID()` and `GetWindowResizeBorderID()` for the current window, that would cover cases of resizing with the mouse.
